Libplanet.Store/BaseIndex.cs

```diff
@@ -1,11 +1,11 @@
-#nullable disable
 using System;
 using System.Collections;
 using System.Collections.Generic;
 
 namespace Libplanet.Store
 {
     public abstract class BaseIndex<TKey, TVal> : IDictionary<TKey, TVal>
+        where TKey : notnull
```

Comment:
See #3645. You can leave it as it is for now.

Reply:
I have reverted that code.